test/parallel/test-fs-readdir-types.js

```diff
@@ -57,6 +57,14 @@ fs.readdir(readdirDir, {
   assertDirents(dirents);
 }));
 
+// Check the promisified version
+assert.doesNotReject(async () => {
```

Comment:
The doesNotReject should be obsolete. Any unhandled rejection is going to result in a test failure.

Reply:
I'm OK with leaving `assert.doesNotReject()` in there anyway. In fact, I think I prefer it. It means that someone reading the test doesn't need to know that `common` performs the magic of adding a listener to the `'unhandledRejection'` event.